zerver/tests/test_messages.py

```diff
@@ -63,7 +63,7 @@ def test_topics_history(self):
         self.login(email)
 
         user_profile = get_user_profile_by_email(email)
-        stream = Stream.objects.get(name=stream_name)
+        stream = get_stream(name=stream_name)
```

Comment:
I think `get_stream()` doesn't have `name` argument there.. Let's we look at the definition of the function on `zerver/models.py`
```.py
def get_stream(stream_name, realm):
    # type: (Text, Realm) -> Optional[Stream]
    try:
        return get_stream_backend(stream_name, realm)
    except Stream.DoesNotExist:
        return None
```

Could you change this and all related to it ?